includes/phing/build/help/toolkit.xml

```diff
@@ -67,47 +70,28 @@
             <available file="${project.basedir}/.git/hooks" type="dir" />
             <then>
                 <foreach param="hook-name" absparam="abshook-name" target="toolkit-hooks-git-update-hook">
-                    <fileset dir="${toolkit.dir.incl.git.hooks}">
-                        <type type="dir" />
-                        <depth max="1" />
-                        <depth min="0" />
+                    <fileset dir="${project.basedir}/resources/git/hooks">
+                        <include name="applypatch-msg, commit-msg, post-applypatch, post-checkout, post-commit, post-merge, post-receive, post-rewrite, post-update, pre-applypatch, pre-auto-gc, pre-commit, prepare-commit-msg, pre-push, pre-rebase, pre-receive, push-to-checkout, rebase, sendemail-validate, update" />
+                        <include name="*/*" />
+                        <type type="file" />
+                        <depth max="2" />
+                        <depth min="1" />
                     </fileset>
+                    <mapper type="regexp" from="(.*?)/(.*?)" to="\1"/>
                 </foreach>
             </then>
         </if>
     </target>
 
     <target name="toolkit-hooks-git-update-hook" description="Update a single git hook." hidden="true">
-        <exec command="ls -Am -I .gitkeep ${toolkit.dir.incl.git.hooks}/${hook-name}" outputproperty="toolkit-hook-directory-contents" error="/dev/null" />
-        <exec command="ls -Am -I .gitkeep ${project.basedir}/resources/git/hooks/${hook-name}" outputproperty="project-hook-directory-contents" error="/dev/null" />
-        <php expression='trim("${toolkit-hook-directory-contents}, ${project-hook-directory-contents}", ", ");' level="debug" returnProperty="hook-directory-contents" />
         <if>
-            <and>
-                <istrue value="${hook-directory-contents}" />
-            </and>
+            <available file="${project.basedir}/.git/hooks" type="dir" />
             <then>
-                <exec command="cp ${toolkit.dir.incl.git}/execute.sh ${project.basedir}/.git/hooks/${hook-name}" />
+                <exec command="ls -Am -I .gitkeep ${project.basedir}/resources/git/hooks/${hook-name}" outputproperty="hook-directory-contents" error="/dev/null" />
+                <exec command="rm ${project.basedir}/.git/hooks/${hook-name}" />
+                <exec command="ln -s ../../includes/git/execute.sh ${project.basedir}/.git/hooks/${hook-name}"  />
```

Comment:
`../../includes/git/execute.sh` this will not work. Includes is inside vendor folder not in subsite project.